utility_network/perform-valve-isolation-trace/src/main/java/com/esri/samples/perform_valve_isolation_trace/PerformValveIsolationTraceLauncher.java

```diff
@@ -0,0 +1,11 @@
+package com.esri.samples.perform_valve_isolation_trace;
```

Comment:
Copyright missing

Reply:
Good eye. We don't generally put a copyright in the Launcher class. I wonder if we should?